docs/experimental-features/host-risk-score.asciidoc

```diff
@@ -164,6 +164,11 @@ image::images/data-tables.png[]
 [discrete]
 === Troubleshooting
 
+[sidebar]
+--
+These steps refer to the original host risk score module.
```

Comment:
@natasha-moore-elastic would you have any reservations about changing `original` to `legacy`? We have been referring to it as such internally.

Reply:
@jaredburgettelastic we currently use 'original' and 'latest' in the public docs to refer to what we internally call 'old'/'legacy' and 'new' risk engine. I remember the terminology was discussed with PM when the new risk engine was introduced, but I can’t remember exactly why 'original' and 'latest' were chosen. Maybe @jmikell821 remembers, but she's on PTO this week.

If we did change 'original'  to 'legacy', we’d need to make that change in all instances where we use it in the docs. My only concern is that changing the terminology now could introduce confusion for users if they are used to seeing the terms that we currently use.